src/frontend/XAPI/XAPIStatement.ts

```diff
@@ -77,8 +80,42 @@ export const verbDefinition: {
 };
 
 export class XAPIStatement {
+  static toFixed(rawNumber: number, length: number = 3): number {
+    return parseFloat(rawNumber.toFixed(length));
+  }
+
+  private startSegments: number[] = [];
+  private endSegments: number[] = [];
+  // tslint:disable-next-line:variable-name
+  private _duration: number = 0;
+
   constructor(private jwt: string, private sessionId: string) {}
 
+  set duration(duration: number) {
```

Comment:
imo this is not idiomatic javascript. Can we just make this a function call please?